spec/requests/atmosphere/api/user_keys_spec.rb

```diff
@@ -94,23 +94,23 @@
   end
 
   describe 'POST /user_keys' do
+    let(:new_key_request) do
+      {
+        user_key: {
+          name: 'new_user_key',
+          public_key: user_key1.public_key
+        }
+      }
+    end
+
     context 'when unauthenticated' do
       it 'returns 401 Unauthorized error' do
-        post api("/user_keys")
+        post api("/user_keys"), new_key_request
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.